specification/spec.md

```diff
@@ -0,0 +1,113 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# BIDS Stats Models Specification
+
+Editors:
+: [Alejandro de la Vega][] ([University of Texas at Austin][])
+: [Christopher J Markiewicz][] ([Stanford University][])
+
+---
+
+## Abstract
+
+This document contains specifications for writing BIDS Stats Models, a recipe for fitting
+hierarchical statistical models to neuroimaging datasets.
+This specification was developed in the [Brain Imaging Data Structure (BIDS)][BIDS] community
+as BIDS Extension Proposal (BEP) 2.
+
+## Status of this document
+
+The previously released version of this specification was [1.0.0rc0][].
+
+## Introduction
+
+The components of this specification interact,
+and it is difficult to discuss one structure without referencing others.
+Therefore this document makes no attempt to introduce all concepts before referring to them.
+For a more graduated introduction of these concepts, please refer to the [Walkthrough](walkthrough-intro).
+
+## On-disk layout
+
+BIDS Stats Models are [JSON][] ([RFC8259][]) documents containing the following fields:
+
+* {py:attr}`~bsmschema.models.BIDSStatsModel.Name`
+* {py:attr}`~bsmschema.models.BIDSStatsModel.BIDSModelVersion`
+* {py:attr}`~bsmschema.models.BIDSStatsModel.Description`
+* {py:attr}`~bsmschema.models.BIDSStatsModel.Input`
+* {py:attr}`~bsmschema.models.BIDSStatsModel.Nodes`
+* {py:attr}`~bsmschema.models.BIDSStatsModel.Edges`
+
+An example top-level structure has the form:
+
+```YAML
+{
+  "Name": "my_first_model",
+  "BIDSModelVersion": "1.0",
+  "Input": {
+    "task": "motor"
+  },
+  "Description": "My first BIDS model: a simple 2-condition contrast.",
+  "Nodes": [
+    {
```

Comment:
Do we need this level of detail in a skeleton example? 

I'd say either link to a full example, or keep it a super high level skeleton (i.e. only top level keys)